src/zope/schema/tests/test_fieldproperty.py

```diff
@@ -283,7 +283,7 @@ def test_field_event(self):
         self.assertEqual(len(log), 6)
         event = log[0]
         self.assertTrue(isinstance(event, FieldUpdatedEvent))
```

Comment:
It seems like it would be good to add a call to `zope.interface.verify.verifyObject` to make sure there are no future regressions:
```python
self.assertTrue(verify.verifyObject(IFieldUpdatedEvent, event))
```

Reply:
Currently, by the way, that fails like this:
```python traceback
>>> verify.verifyObject(interfaces.IFieldUpdatedEvent, FieldUpdatedEvent(None, None, None, None))
Traceback (most recent call last):
  File "//zope/interface/verify.py", line 99, in _verify_element
    attr = getattr(candidate, name)
AttributeError: 'FieldUpdatedEvent' object has no attribute 'object'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "//zope/interface/verify.py", line 172, in verifyObject
    return _verify(iface, candidate, tentative, vtype='o')
  File "//zope/interface/verify.py", line 91, in _verify
    raise excs[0]
  File "//zope/interface/verify.py", line 85, in _verify
    _verify_element(iface, name, desc, candidate, vtype)
  File "//zope/interface/verify.py", line 106, in _verify_element
    raise BrokenImplementation(iface, desc, candidate)
zope.interface.exceptions.BrokenImplementation: The object <zope.schema.fieldproperty.FieldUpdatedEvent object at 0x100a9c780> has failed to implement interface <InterfaceClass zope.schema.interfaces.IFieldUpdatedEvent>: The zope.schema.interfaces.IFieldEvent.object attribute was not provided.
```

Reply:
Done